packages/adblocker/test/engine/engine.test.ts

```diff
@@ -493,24 +493,28 @@ $csp=baz,domain=bar.com
   });
 
   describe('network filters', () => {
-    // Collect all filters from all requests in the dataset. Each test case
-    // contains one request as well as a list of filters matching this request
-    // (exceptions, normal filters, etc.). We create a big list of filters out
-    // of them.
-    const allRequestFilters = requests.map(({ filters }) => filters.join('\n')).join('\n');
-
-    // Create several base engines to be used in different scenarii:
-    // - Engine with *no filter* optimizations *enabled*
-    // - Engine with *no filter* optimizations *disabled*
-    // - Engine with *all filters* optimizations *enabled*
-    // - Engine with *all filters* optimizations *disabled*
-    const engineFullOptimized = Engine.parse(allRequestFilters, {
-      debug: true,
-      enableOptimizations: true,
-    });
-    const engineFull = Engine.parse(allRequestFilters, {
-      debug: true,
-      enableOptimizations: false,
+    let engineFullOptimized: Engine;
```

Comment:
those has been moved from describe top level into `before` so that it is lazily initialised